src/init.ts

```diff
@@ -115,13 +118,13 @@ export async function init(signerService: SignerService) {
       assetsService,
       connectToDidRegistry,
       stakingPoolService,
+      verifiableCredentialsService,
```

Comment:
@JGiter Were you able to update the `quick_start` doc with this new service? Or were you thinking we should do it in a separate story/task? (Or did I just miss it in the changes 😅 )

Reply:
Updated now. I skipped updating before because I thought you removed documentation suggestion about verifiable credential service. Probably it was about storage client